internal/core/command/rest_device.go

```diff
@@ -96,7 +88,9 @@ func issueDeviceCommand(
 		return
 	}
 
-	w.Header().Set(clients.ContentType, clients.ContentTypeJSON)
+	// Set the returned header Content-type based on header Content-type received in
+	// the Device Service request (No need to inspect it).
+	w.Header().Set(clients.ContentType, r.Header.Get(clients.ContentType))
```

Comment:
This functionality does not match the intent of the comment above. Here we are copying the `Content-type` header from the original request, not the device service's response. In our meeting we discussed this and why this is not the solution we are after. To achieve what we are after we need to get the headers from the response from the device service's response, which at this point, only exists in the underlying operator:
https://github.com/edgexfoundry/edgex-go/blob/93a900b48c21df2a0611ee8b9bdbe214b0c3fa76/internal/core/command/device.go#L121

It seems like it is becoming hard to track which request is which since we have 2 in the context of this operation. Can we properly name each request to make it more obvious what is going on? Using `r`, `req`, or `request` does not help the situation. I think naming the request received by core-command to something like `originalRequest` , and the request to device service's to something like `deviceServiceProxiedRequest` will make these types of inconsistencies more obvious.